app/register/route.ts

```diff
@@ -18,4 +19,9 @@ export default class Register extends Route {
     model() {
         return this.store.createRecord('user-registration');
     }
+
+    setupController(controller: RegisterController, model: any, transition: Transition) {
+        super.setupController(controller, model, transition);
+        controller.setup();
+    }
```

Comment:
Is this necessary now? This seems like something that would just happen.

Reply:
So this is a little confusing to look at now as `setup` isn't an actual controller lifecycle hook (it was a [hook that ember-parachute added](https://github.com/offirgolan/ember-parachute#usage)). I tried using the controller's `init` hook to do some of that provider-fetching stuff, but the QPs weren't defined in at the `init` stage, so this was the next easiest solution I could think of. I could change the `setup` function name to something else to be a potentially less misleading?

Reply:
Ah, I see.